docs/orleans/implementation/testing.md

```diff
@@ -155,32 +50,28 @@ public new virtual IGrainFactory GrainFactory
 Now we can create our grain outside of the Orleans runtime and use mocking to control the behavior of `GrainFactory`:
 
 ```csharp
-using System;
-using System.Threading.Tasks;
-using Orleans;
 using Xunit;
 using Moq;
 
-namespace Tests
+namespace Tests;
+
+public class WorkerGrainTests
 {
-    public class WorkerGrainTests
+    [Fact]
+    public async Task RecordsMessageInJournal()
     {
-        [Fact]
-        public async Task RecordsMessageInJournal()
-        {
-            var data = "Hello, World";
-            var journal = new Mock<IJournalGrain>();
-            var worker = new Mock<WorkerGrain>();
-            worker
-                .Setup(x => x.GrainFactory.GetGrain<IJournalGrain>(It.IsAny<Guid>()))
-                .Returns(journal.Object);
-
-            await worker.DoWork(data)
-
-            journal.Verify(x => x.Record(data), Times.Once());
-        }
+        var data = "Hello, World";
+        var journal = new Mock<IJournalGrain>();
+        var worker = new Mock<WorkerGrain>();
+        worker
+            .Setup(x => x.GrainFactory.GetGrain<IJournalGrain>(It.IsAny<Guid>()))
+            .Returns(journal.Object);
+
+        await worker.DoWork(data)
+
+        journal.Verify(x => x.Record(data), Times.Once());
     }
 }
 ```
 
-Here we create our grain under test, `WorkerGrain`, using Moq which means we can then override the behavior of the `GrainFactory` so that it returns a mocked `IJournalGrain`. We can then verify that our `WorkerGrain` interacts with the `IJournalGrain` as we expect.
+Here we create our grain under test, `WorkerGrain`, using Moq that means we can then override the behavior of the `GrainFactory` so that it returns a mocked `IJournalGrain`. We can then verify that our `WorkerGrain` interacts with the `IJournalGrain` as we expect.
```

Comment:
```suggestion
Here you create the grain under test, `WorkerGrain`, using Moq, which means you can override the behavior of the `GrainFactory` so that it returns a mocked `IJournalGrain`. You can then verify that the `WorkerGrain` interacts with the `IJournalGrain` as you expect.
```